rbac/management/relation_replicator/relation_replicator.py

```diff
@@ -55,6 +55,7 @@ class ReplicationEventType(str, Enum):
     MIGRATE_SYSTEM_ROLE_ASSIGMENT = "migrate_system_role_assignment"
     APPROVE_CROSS_ACCOUNT_REQUEST = "approve_cross_account_request"
     EXPIRE_CROSS_ACCOUNT_REQUEST = "expire_cross_account_request"
+    MIGRATE_ACCOUNT_REQUEST = "migrate_cross_account_request"
```

Comment:
Minor nit for consistency:

```suggestion
    MIGRATE_CROSS_ACCOUNT_REQUEST = "migrate_cross_account_request"
```